package/toltec-bootstrap/entware-reenable

```diff
@@ -31,6 +34,42 @@ UNIT
     systemctl enable --now opt.mount 2> /dev/null
 }
 
+# Mount /opt/share/zoneinfo to /usr/share/zoneinfo
+timezone-mount() {
+    mkdir -p "$entware_path"/share/zoneinfo
+
+    if [[ ! -f $systemd_timezone_mount_path ]]; then
+        log "Updating timezone.mount location"
+
+        if systemctl -q is-enabled timezone.mount; then
+            # Remove path to old file if still symlinked
+            systemctl disable timezone.mount
+        fi
+    fi
```

Comment:
Why is this needed?

Reply:
In case we change the location. I copied from the entware-mount method.

Reply:
Assuming we changed the location in the future, this would not remove the old file (as opposed to what is done in `entware-mount`). I propose we wait until we actually change the location to add this block, then we’ll know what the old name is and will be able to clean it.

Reply:
Sounds good